### DISCUSSION TITLE
Form inputs not binding to property updates

### DISCUSSION THREAD

**Discussion by PhillipMwaniki** [livewire/livewire#6689] [1. Help] [answered]
Hello, I have an issue with Livewire 3. My input fields in the form do not update should their model update. I followed all instructions on installation and setting up the Livewire components
I've tried creating other projects with Livewire and the result is the same. I noticed that the issue keeps reoccurring with form inputs.
I also switched from valet to Herd thinking that it's a server issue.

https://github.com/livewire/livewire/assets/2037708/ae33cfcb-9300-455f-b123-ea72949b00ff

**Comment by joshhanley** [comment]
@PhillipMwaniki V2 or V3? Also hard to say without seeing any code. Also are there any browser console errors?

**Comment by PhillipMwaniki** [comment, marked as answer]
Hey,
Fixed the issue. Breeze is the culprit. You have to go to resources/js/app.js and delete the Alpine import over there.
Apparently Livewire 3 ships with a copy of Alpine and so does Breeze. 
<img width="476" alt="image" src="https://github.com/livewire/livewire/assets/2037708/19753969-2af3-48a8-a7e2-4dcbf5e6e893">
